### PR TITLE
Modify nfv cases to support both q35 and pc machine_type

### PR DESCRIPTION
**Pull request by yafu-1**
Signed-off-by: Yan Fu <yafu@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/sriov/sriov.py [hunk 1]
```diff
@@ -717,12 +717,14 @@ def remove_devices(vmxml, device_type):
...
             utils_test.libvirt.check_exit_status(result, expected_error)
         if nfv:
-            for os_machine_type in (machine_type, vmxml.os.machine):
-                'q35' in os_machine_type or test.cancel("nfv only run with q35 machine type")
             vf_driver = os.readlink(os.path.join(pci_device_dir, vf_list[0], "driver")).split('/')[-1]
             vmxml.remove_all_device_by_type('controller')
             remove_devices(vmxml, 'address')
             remove_devices(vmxml, 'usb')
+            osxml = vmxml.os
+            if "i440fx" in vmxml.os.machine:
```
Comment (kylazhang) [inline libvirt/tests/src/sriov/sriov.py]: what about "pc" ?

File: libvirt/tests/cfg/sriov/sriov.cfg [hunk 2]
```diff
@@ -56,7 +56,6 @@
                     no scalability_test, vf_pool
                 - nop:
                 - nfv:
-                    only q35
```
Comment (kylazhang) [inline libvirt/tests/cfg/sriov/sriov.cfg]: so all the architecture will support? what about only i440fx, q35?

**Comment by chloerh** [comment]
LGTM

File: libvirt/tests/src/sriov/sriov.py (hunk 1 above)
Comment (yafu-1) [reply libvirt/tests/src/sriov/sriov.py]: q35 machine type also contains 'pc', such as: pc-q35-rhel7.6.0

File: libvirt/tests/cfg/sriov/sriov.cfg (hunk 2 above)
Comment (yafu-1) [reply libvirt/tests/cfg/sriov/sriov.cfg]: No, only support i440fx and q35. I will modify the cfg file.

File: libvirt/tests/src/sriov/sriov.py (hunk 1 above)
Comment (kylazhang) [reply libvirt/tests/src/sriov/sriov.py]: I mean the machine_type='pc' which means default/alias of i440fx

Comment (kylazhang) [reply libvirt/tests/src/sriov/sriov.py]: I know it. Thanks

**Pull request merged**